SlashGaming-Diablo-II-Free-Resolution/src/patches/interface_bar/d2client_enable_800_new_stats_button_patch/d2client_enable_800_new_stats_button_patch_1_09d.cc

```diff
@@ -470,6 +580,16 @@ Enable800NewStatsButtonPatch_1_09D::GetPatchAddressAndSize12() {
           0x48B81 - 0x48B5C
       );
     }
+
+    case ::d2::GameVersion::k1_10: {
+      return PatchAddressAndSize(
+          ::mapi::GameAddress::FromOffset(
+              ::d2::DefaultLibrary::kD2Client,
+              0x4F5E7
```

Comment:
While this patch location starts two instructions up, it ends on the same relative places, and performs the same thing.